spec/helpers/vm_helper_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "spec_helper"
+
+# @record.operating_system && @record.operating_system.processes.maximum(:updated_on)
+
+describe VmHelper do
+  describe "#last_date_processes" do
+    it "supports vm without an OS" do
```

Comment:
@kbrock I think you have the "it's" backwards.  This one has an OS.  Then next does not.